pkg/instill/main.go

```diff
@@ -62,18 +62,18 @@ type GetModelRes struct {
 }
 
 type Model struct {
-	Name            string    `json:"name"`
-	UID             string    `json:"uid"`
-	ID              string    `json:"id"`
-	Description     string    `json:"description"`
-	ModelDefinition string    `json:"model_definition"`
-	Configuration   any       `json:"configuration"`
-	Task            string    `json:"task"`
-	State           string    `json:"state"`
-	Visibility      string    `json:"visibility"`
-	User            string    `json:"user"`
-	CreateTime      time.Time `json:"create_time"`
-	UpdateTime      time.Time `json:"update_time"`
+	Name            string           `json:"name"`
+	UID             string           `json:"uid"`
+	ID              string           `json:"id"`
+	Description     string           `json:"description"`
+	ModelDefinition string           `json:"model_definition"`
+	Configuration   any              `json:"configuration"`
+	Task            connectorPB.Task `json:"task"`
```

Comment:
The json conversion will be wrong here, need further data conversion.

Reply:
updated